tests/test_sessions.py

```diff
@@ -0,0 +1,75 @@
+import os
+import tempfile
+import pytest
+import json
+from flasksrc import create_app, db
+from flask_jwt_extended import create_access_token
+
+@pytest.fixture
+def client():
+    app = create_app({'TESTING': True})
+    db_fd, app.config['DATABASE'] = tempfile.mkstemp()
+
+    with app.test_client() as client:
+        with app.app_context():
+            db.init_db()
+        yield client
+
+    os.close(db_fd)
+    os.unlink(app.config['DATABASE'])
+def test_topo_id_post(client):
+    """Check the POST topo_id returns a session id if valid or error if not"""
+    client.post(
+          '/api/user',
+          data={
+              'username': 'Arthur@projectreclass.org',
+              'password': 'BaLtH@$0R',
+              'first_name': 'Arthur',
+          },
+    )
+    url = "/api/toynet/session"
+
+    pv = ({'toynet_topo_id': 1, 'toynet_user_id': 'Arthur@projectreclass.org',})
```

Comment:
lowercase `Arthur` to `arthur` for consistency with other test files

Reply:
updated